src/RegistrationMsg.chpl

```diff
@@ -542,7 +542,7 @@ module RegistrationMsg
         }
 
         // type possibilities for pdarray and strings
-        var simpleTypes: list(string) = ["pdarray","int64", "uint8", "uint64", "float64", "bool", "strings", "string", "str"];
+        var simpleTypes: list(string) = ["pdarray","int64", "uint8", "uint64", "float64", "bool", "strings", "string", "str", "segarray"];
```

Comment:
Since we have used this list in multiple places in the code, make it a Global so we don't redefine it multiple times.

Reply:
Done